azure-pipelines.yaml

```diff
@@ -0,0 +1,86 @@
+# Starter pipeline
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+name: $(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
```

Comment:
We can use `branches` field under `trigger` for a clear syntax

eg: 
<img width="178" alt="image" src="https://github.com/DEFRA/ncea-frontend/assets/131792244/b5e55a45-339c-4cbd-adf2-26ed02c38098">

Reply:
Done